src/devices/paulmann.ts

```diff
@@ -176,6 +176,18 @@ const definitions: Definition[] = [
             'on', 'off', 'toggle', 'brightness_step_up', 'brightness_step_down', 'color_temperature_move', 'color_move', 'brightness_stop',
             'brightness_move_down', 'brightness_move_up', 'color_loop_set', 'enhanced_move_to_hue_and_saturation', 'scene_*'])],
     },
+    {
+        fingerprint: [{ modelID: 'RGBW', manufacturerName: 'Paulmann Licht GmbH' }],
```

Comment:
No need to add a fingerprint since there is already a `zigbeeModel`, please remove it and check if everything still works. If not, provide me the data/database.db entry of this device.

Reply:
Could not get it to run without fingerprint.
DB entry of one of these bulbs (there are different shapes and sizes of these bulbs, but they all report to z2m the same way):
```
{
        "field1": "{id:54",
        "field10": "242]",
        "field100": null,
        "field101": null,
        "field102": null,
        "field103": null,
        "field104": null,
        "field105": null,
        "field106": null,
        "field107": null,
        "field108": null,
        "field109": null,
        "field11": "endpoints:{1:{profId:260",
        "field110": null,
        "field111": null,
        "field112": null,
        "field12": "epId:1",
        "field13": "devId:269",
        "field14": "inClusterList:[0",
        "field15": "3",
        "field16": "4",
        "field17": "5",
        "field18": "6",
        "field19": "8",
        "field2": "type:Router",
        "field20": "768",
        "field21": "4096",
        "field22": "4111]",
        "field23": "outClusterList:[25]",
        "field24": "clusters:{genBasic:{attributes:{modelId:RGBW",
        "field25": "manufacturerName:Paulmann Licht GmbH",
        "field26": "powerSource:1",
        "field27": "zclVersion:3",
        "field28": "appVersion:0",
        "field29": "stackVersion:2",
        "field3": "ieeeAddr:0xa4c13800ac4921fa",
        "field30": "hwVersion:146",
        "field31": "swBuildId:PIIC4200}}",
        "field32": "lightingColorCtrl:{attributes:{colorCapabilities:31",
        "field33": "colorTempPhysicalMin:153",
        "field34": "colorTempPhysicalMax:454}}",
        "field35": "genOnOff:{attributes:{onOff:1}}}",
        "field36": "binds:[{cluster:6",
        "field37": "type:endpoint",
        "field38": "deviceIeeeAddress:0x00212effff07abab",
        "field39": "endpointID:1}]",
        "field4": "nwkAddr:33633",
        "field40": "configuredReportings:[{cluster:6",
        "field41": "attrId:0",
        "field42": "minRepIntval:0",
        "field43": "maxRepIntval:3600",
        "field44": "repChange:0}]",
        "field45": "meta:{}}",
        "field46": "242:{profId:41440",
        "field47": "epId:242",
        "field48": "devId:97",
        "field49": "inClusterList:[]",
        "field5": "manufId:4632",
        "field50": "outClusterList:[33]",
        "field51": "clusters:{}",
        "field52": "binds:[]",
        "field53": "configuredReportings:[]",
        "field54": "meta:{}}}",
        "field55": "appVersion:0",
        "field56": "stackVersion:2",
        "field57": "hwVersion:146",
        "field58": "swBuildId:PIIC4200",
        "field59": "zclVersion:3",
        "field6": "manufName:Paulmann Licht GmbH",
        "field60": "interviewCompleted:true",
        "field61": "meta:{configured:88764544}",
        "field62": "lastSeen:1693940455214",
        "field63": "defaultSendRequestWhen:immediate}",
        "field64": null,
        "field65": null,
        "field66": null,
        "field67": null,
        "field68": null,
        "field69": null,
        "field7": "powerSource:Mains (single phase)",
        "field70": null,
        "field71": null,
        "field72": null,
        "field73": null,
        "field74": null,
        "field75": null,
        "field76": null,
        "field77": null,
        "field78": null,
        "field79": null,
        "field8": "modelId:RGBW",
        "field80": null,
        "field81": null,
        "field82": null,
        "field83": null,
        "field84": null,
        "field85": null,
        "field86": null,
        "field87": null,
        "field88": null,
        "field89": null,
        "field9": "epList:[1",
        "field90": null,
        "field91": null,
        "field92": null,
        "field93": null,
        "field94": null,
        "field95": null,
        "field96": null,
        "field97": null,
        "field98": null,
        "field99": null
    }
```

Reply:
Then drop the `zigbeeModel` part

Reply:
done, works that way now